src/store/dashboard/awsCloudDashboard/awsCloudDashboardWidgets.ts

```diff
@@ -19,7 +19,7 @@ export const computeWidget: AwsCloudDashboardWidget = {
   reportPathsType: ReportPathsType.awsCloud,
   reportType: ReportType.instanceType,
   details: {
-    costKey: 'aws_cloud_dashboard.cost_label',
+    costKey: 'chart.cost',
```

Comment:
This really doesn't have anything to do with charts, but the details section of the dashboard card